src/notehead.ts

```diff
@@ -133,6 +133,10 @@ export class NoteHead extends Note {
     this.glyph = Tables.getGlyphProps(this.duration, this.noteType);
     defined(this.glyph, 'BadArguments', `No glyph found for duration '${this.duration}' and type '${this.noteType}'`);
 
+    // Swap out the glyph with leger lines
+    if (this.isRest() && (this.line >= 6 || this.line < 0) && this.glyph.leger_code_head) {
+      this.glyph.code_head = this.glyph.leger_code_head;
+    }
     this.glyph_code = this.glyph.code_head;
```

Comment:
Hi @AaronDavidNewman I realised that you did not add the else branch as suggested above. Is there a reason for that or it was just missed?

Reply:
I did miss the request.  It seems wrong to me though - `this.glyph_code` should match `this.glyph.code_head`.  I think it would be confusing if they were different.  `this.glyph_code` is legacy, and is sort of redundant anyway.  It seems we could always use `this.glyph` properties.

I think it is confusing because `this.glyph` is not a `Glyph` instance.  It is just a POJO used to create the actual glyph.  And the glyph properties from Tables are based on the duration, but the pitch of the note is not known.

I did see one possible issue though - `this.line` should consider the actual number of lines in the staff, if they are known.  Can't assume they are always 5.  But I'm not sure if the staff is always known here, either.